.github/workflows/terraform.yml

```
@@ -202,15 +272,30 @@ jobs:
         working-directory: terraform
         run: |
-          # Import existing DynamoDB tables if they exist
-          terraform import aws_dynamodb_table.iphone_prices iphone_prices || true
-          terraform import aws_dynamodb_table.official_prices official_prices || true
-          terraform import aws_dynamodb_table.price_history price_history || true
-          terraform import aws_dynamodb_table.price_predictions price_predictions || true
+          # Function to check if resource exists in state
+          check_state() {
+            terraform state list | grep -q "$1" && echo "true" || echo "false"
+          }
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)